src/Discord.Net.Core/Entities/Activities/SpotifyGame.cs

```diff
@@ -31,6 +31,23 @@ public class SpotifyGame : Game
         ///     A string containing the name of the song (e.g. <c>Lonely Together (feat. Rita Ora)</c>).
         /// </returns>
         public string TrackTitle { get; internal set; }
+
+        /// <summary>
+        ///     Gets the date the track began playing at.
```

Comment:
`Gets the date when the track started playing` sounds a little better.

Reply:
Thanks, fixed with https://github.com/discord-net/Discord.Net/pull/1414/commits/2f9e286c09116022692d7344227c708815d4bbfb